.travis.yml

```diff
@@ -0,0 +1,29 @@
+sudo: required
+
```

Comment:
We might need to add 
```
jdk:
 - oraclejdk8 # openJDK crashes sometimes; tensorframes is compiled w.r.t JDK8
```
following TensorFrames Travis settings https://github.com/databricks/tensorframes/blob/master/.travis.yml